packages/tempus/src/Tempus_IntegratorAdjointSensitivity_decl.hpp

```diff
@@ -179,31 +186,51 @@ class IntegratorAdjointSensitivity :
     const Teuchos::RCP<const SolutionHistory<Scalar> >& state_solution_history,
     const Teuchos::RCP<const SolutionHistory<Scalar> >& adjoint_solution_history);
 
-  Teuchos::RCP<Thyra::ModelEvaluator<Scalar> > model_;
-  Teuchos::RCP<AdjointAuxSensitivityModelEvaluator<Scalar> > adjoint_model_;
-  Teuchos::RCP<IntegratorBasicOld<Scalar> > state_integrator_;
-  Teuchos::RCP<IntegratorBasicOld<Scalar> > adjoint_integrator_;
-  Teuchos::RCP<SolutionHistory<Scalar> > solutionHistory_;
+  Teuchos::RCP<Thyra::ModelEvaluator<Scalar>> model_;
+  Teuchos::RCP<IntegratorBasic<Scalar>> state_integrator_;
+  Teuchos::RCP<AdjointAuxSensitivityModelEvaluator<Scalar>> adjoint_model_;
+  Teuchos::RCP<IntegratorBasic<Scalar>> adjoint_integrator_;
+  Teuchos::RCP<SolutionHistory<Scalar>> solutionHistory_;
   int p_index_;
   int g_index_;
   bool g_depends_on_p_;
   bool f_depends_on_p_;
   bool ic_depends_on_p_;
   bool mass_matrix_is_identity_;
-  Teuchos::RCP<const Thyra::MultiVectorBase<Scalar> > dxdp_init_;
-  Teuchos::RCP<Thyra::MultiVectorBase<Scalar> > dgdp_;
+  Teuchos::RCP<const Thyra::MultiVectorBase<Scalar>> dxdp_init_;
+  Teuchos::RCP<Thyra::MultiVectorBase<Scalar>> dgdp_;
 };
 
 /// Nonmember constructor
-template<class Scalar>
-Teuchos::RCP<IntegratorAdjointSensitivity<Scalar> >
+/**
+ * @brief Nonmember constructor
+ *
+ * This nonmember constructor calls parses the `pList` provided to constructor
+ * the sub-objects needed to call the full `IntegratorAdjointSensitivity`
+ * construtor
+ *
+ * @param pList  ParameterList defining the integrator options and options
+ *               defining the sensitivity analysis
+ * @param model  ModelEvaluator for the problem
+ *
+ * @return Time integrator implementing adjoint sensitivity
+ */
+template <class Scalar>
+Teuchos::RCP<IntegratorAdjointSensitivity<Scalar>>
 integratorAdjointSensitivity(
```

Comment:
We have the convention of prepending "create" to nonmember constructors with camel case, e.g.,  integratorAdjointSensitivity() --> creatIntegratorAdjointSensitivity().